src/IR.h

```diff
@@ -909,6 +909,16 @@ struct Prefetch : public StmtNode<Prefetch> {
     static const IRNodeType _node_type = IRNodeType::Prefetch;
 };
 
+struct HoistedStorage : public StmtNode<HoistedStorage> {
```

Comment:
Needs docs. Not sure what exactly this IR node means. I think it's the location where storage will be hoisted to after storage flattening for the named Func/realize node?